cards/src/main/resources/cards/custom/group6/witch_doctor/minion_hexcrazed_vessel.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "Hexcrazed Vessel",
+  "baseManaCost": 6,
+  "type": "MINION",
+  "heroClass": "ROSE",
+  "baseAttack": 6,
+  "baseHp": 5,
+  "rarity": "RARE",
+  "description": "After you cast a spell on this, add a random Witch Doctor spell to your hand.",
+  "trigger": {
+    "eventTrigger": {
+      "class": "TargetAcquisitionTrigger",

```

Comment:
same thing, should probably be `AfterSpellCastedTrigger`

Reply:
Same as above